src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ContentService.cs

```diff
@@ -156,7 +156,8 @@ private static JsonSerializerSettings GetJsonSerializerSettings(NamingStrategy n
                 ContractResolver = new DefaultContractResolver
                 {
                     NamingStrategy = namingStrategy
-                }
+                },
+                TypeNameHandling = TypeNameHandling.Auto
```

Comment:
What does this do?

Reply:
In publisher for deserialising the polymorphic types `IContentBlockChart `and `IContentBlockViewModel `I was trying not to use the converter `ContentBlockChartConverter `and introduce another converter similar to `ContentBlockConverter `which I don't think should be necessary.  Newtonsoft.Json can handle serialisation of polymorphic types either automatically by adding a `$type` property where necessary (`TypeNameHandling.Auto`) or explicitly by adding `$type` to every object  (`TypeNameHandling.All`).